libs/foo/src/executors/bar/combine.js

```diff
@@ -0,0 +1,32 @@
+module.exports = async function* combine(iterable) {
```

Comment:
yanked from https://stackoverflow.com/a/50586391

just a copy-paste for combine AsyncIterators to exhaust them concurrently